src/components/Comments/ReplyCommentItem.js

```diff
@@ -272,42 +306,71 @@ const ReplyCommentItem = ({
-              <MemoCommentReply />
-              <Text style={styles.btnReplyText}>Reply</Text>
-            </ButtonHightlight>
-          </TouchableOpacity>
+          <>
+            <TouchableOpacity testID="replyBtn" activeOpacity={1} onPress={onPress}>
+              <ButtonHightlight
+                onLongPress={handleLongPress}
+                style={[
+                  styles.btnReply,
+                  {
+                    marginRight: 0,
+                    paddingRight: 0
+                  }
+                ]}
+                onPress={onPress}>
+                <MemoCommentReply />
+              </ButtonHightlight>
+            </TouchableOpacity>
+            {!comment?.is_you && (
+              <TouchableOpacity onPress={onPressDm} testID="sendDMbtn" activeOpacity={1}>
+                <ButtonHightlight
+                  onLongPress={handleLongPress}
+                  onPress={onPressDm}
+                  style={[
+                    styles.btnBlock(comment.user.id === yourselfId),
+                    styles.btn,
+                    {
+                      marginRight: 0,
+                      paddingRight: 0
+                    }
+                  ]}>
+                  <MemoSendDM />
+                </ButtonHightlight>
+              </TouchableOpacity>
+            )}
+          </>
         )}
         {!comment?.is_you && (
           <TouchableOpacity onPress={() => onBlock(comment)} testID="btnBlock" activeOpacity={1}>
             <ButtonHightlight
               onLongPress={handleLongPress}
               onPress={() => onBlock(comment)}
-              style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}>
+              style={[
+                styles.btnBlock(comment.user.id === yourselfId),
+                styles.btn,
+                {
+                  marginRight: 16,
+                  paddingRight: 0
+                }
+              ]}>
               <IconEn name="block" size={15.02} color={COLORS.balance_gray} />
             </ButtonHightlight>
           </TouchableOpacity>
         )}
-        <TouchableOpacity
-          onPress={() => refSheet.current.open()}
-          testID="sendDMbtn"
-          activeOpacity={1}>
-          <ButtonHightlight
-            onLongPress={handleLongPress}
-            onPress={() => refSheet.current.open()}
-            style={[styles.btnBlock(comment.user.id === yourselfId), styles.btn]}>
-            <MemoSendDM />
-          </ButtonHightlight>
-        </TouchableOpacity>
 
         <TouchableOpacity onPress={onDownVote} testID="downvoteBtn">
           <ButtonHightlight
             onLongPress={handleLongPress}
-            style={[styles.arrowup, styles.btn]}
+            style={[
+              styles.arrowup,
+              styles.btn,
+              {
+                paddingLeft: 0,
+                marginLeft: 0,
+                marginRight: 8,
+                paddingRight: 0
+              }
+            ]}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The UI for reply, send DM, and block actions have been updated. Ensure that these changes don't break the layout on different screen sizes and orientations. Also, make sure that the touchable areas for these actions are large enough for good usability.



<!-- This is an auto-generated comment by OSS CodeRabbit -->